src/app/modules/alerts/store/alert.effects.ts

```diff
@@ -4,27 +4,27 @@ import { Store } from '@ngrx/store';
 import { TranslateService } from '@ngx-translate/core';
 import { EMPTY, forkJoin, of } from 'rxjs';
 import {
-  catchError, filter, map, mergeMap, pairwise, switchMap, withLatestFrom,
+  catchError, map, mergeMap, pairwise, switchMap, withLatestFrom,
 } from 'rxjs/operators';
 import { IncomingApiMessageType } from 'app/enums/api-message-type.enum';
 import {
   dismissAlertPressed, dismissAllAlertsPressed,
   reopenAlertPressed,
   reopenAllAlertsPressed,
-  alertAdded,
-  alertChanged,
   alertRemoved,
   alertsLoaded,
   alertsNotLoaded,
+  alertReceived,
 } from 'app/modules/alerts/store/alert.actions';
 import { AlertSlice, selectDismissedAlerts, selectUnreadAlerts } from 'app/modules/alerts/store/alert.selectors';
 import { WebSocketService } from 'app/services/ws.service';
 import { adminUiInitialized } from 'app/store/admin-panel/admin.actions';
+import { alertIndicatorPressed } from 'app/store/topbar/topbar.actions';
 
 @Injectable()
 export class AlertEffects {
   loadAlerts$ = createEffect(() => this.actions$.pipe(
-    ofType(adminUiInitialized),
+    ofType(adminUiInitialized, alertIndicatorPressed, alertReceived),
```

Comment:
This is different from loading alerts when alerts panel open. This will query `alert.list` every time there is a new alert or an update to an existing one.
If we want to still do this, I feel like the larger refactoring is needed, e.g. it may be necessary to trigger this update behaviour only when panel is open.

Reply:
@undsoft 
I agree, but on the other hand - we want to know a `fresh counter` and for this we always need to grab a list of alerts I assume.
<img width="61" alt="Screenshot 2023-08-11 at 14 22 51" src="https://github.com/truenas/webui/assets/22980553/2a9df0dd-1e0f-475d-9217-e7efb2d210ce">

Reply:
or we may handle counter with subscription and events we receive somehow.
And only fetch fresh alerts if the `Alerts panel` is opened and new event received.